examples/classification/plot_lda_qda.py

```diff
@@ -114,9 +113,8 @@ def plot_ellipse(splot, mean, cov, color):
     angle = 180 * angle / np.pi  # convert to degrees
     # filled Gaussian at 2 standard deviation
     ell = mpl.patches.Ellipse(mean, 2 * v[0] ** 0.5, 2 * v[1] ** 0.5,
-                              180 + angle, facecolor=color,
-                              edgecolor='yellow',
-                              linewidth=2, zorder=2)
+                              180 + angle, facecolor=color, alpha=1,
+                              edgecolor='grey', linewidth=2)
     ell.set_clip_box(splot.bbox)
     ell.set_alpha(0.5)
```

Comment:
This sets the alpha. Do try reducing it a little.

Reply:
As I mentioned, tuning this `alpha` doesn't change the rendered figure, a potential bug of matplotlib

Reply:
But the image obviously shows the ellipse as transparent. What is causing that transparency if not this line? Are you saying that `set_alpha(1.0)` leaves it translucent?

Reply:
I set `facecolor="none"`

Reply:
my mistake. I missed that line